modules/core/src/lib/attribute-transition-manager.js

```diff
@@ -173,27 +173,25 @@ export default class AttributeTransitionManager {
   // get current values of an attribute, clipped/padded to the size of the new buffer
   _getNextTransitionStates(transition, settings) {
     const {attribute} = transition;
-    const {size, offset, normalized} = attribute;
+    const {size, normalized} = attribute;
+    const multiplier = attribute.doublePrecision ? 2 : 1;
 
     let toState;
     if (attribute.constant) {
-      toState = new BaseAttribute(this.gl, {constant: true, value: attribute.value, size, offset});
+      toState = new BaseAttribute(this.gl, {constant: true, value: attribute.value, size});
```

Comment:
Can constant attributes be normalized? Should that be passed here in the `opts`?

Reply:
`attribute.value` is already normalized in https://github.com/uber/deck.gl/blob/master/modules/core/src/lib/base-attribute.js#L86